### PR TITLE
Add hg-git/meta.yaml

### PR DESCRIPTION
**Pull request by paugier**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by paugier** [comment]
I didn't find any license file in https://dev.heptapod.net/mercurial/hg-git/

What should I do?

File: recipes/hg-git/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0f0a774e2f94e305ab69d13a5ea1e7e440b4b03486fc1a30784715fcd2ee6290
+
+build:
+  number: 0
+  skip: True  # [py3k]
```
Comment (npavlovikj) [inline recipes/hg-git/meta.yaml]: I think this recipe can be `noarch`. Please remove the `skip` line, add `noarch: python`, and in the `host` and `run` sections use `python <3`.

**Comment by npavlovikj** [comment]
@paugier , looks like `hg-git` has both python library and binary. Since you are installing the python library only, I would suggest you to name the package/recipe `python-hg-git` instead. This way, the package for the binary can be named `hg-git` in future, and users can easily differentiate them.

**Comment by paugier** [comment]
> looks like hg-git has both python library and binary.

I don't understand? hg-git is a pure-Python Mercurial extension (https://dev.heptapod.net/heptapod/hg-git).  

Can you please tell me what you mean by "the binary" (for hg-git)?

**Comment by npavlovikj** [comment]
@paugier , I saw this, https://dev.heptapod.net/mercurial/hg-git/tree/branch/default#commands, and thus my comment. If the command is valid, can you please also test for it with:
```
test:
  commands:
    - hg help -e hggit
```

**Comment by paugier** [comment]
Oh now I understand! I was confused by the term "binary" while it's only Python files :slightly_smiling_face: 

`hg help -e hggit` would require a modification of the config file `.hgrc` so I hope `hg help -e hggit --config "extensions.hggit="` is going to work.

**Comment by npavlovikj** [comment]
Thanks for modifying the recipe accordingly @paugier - it looks good to me!

And sorry about the confusion earlier - I skimmed over the web page, saw the "Makefile" and "commands" part, and assumed there is a binary that needs to be compiled. But the "Makefile" only has the Python install, so `hg help -e hggit` tests great.

**Comment by paugier** [comment]
Thanks for you help @npavlovikj !

**Pull request merged**